prime-router/docs/azure-logs-audit.md

```diff
@@ -0,0 +1,23 @@
+# Documentation of the Current Available Logs in Azure
+
+| Log Type           | Type                                    | PII/PHI                                     | Location                     | In Splunk | Required in Splunk for ATO | Notes                                                   |
+| :----------------- | :-------------------------------------- | :------------------------------------------ | :--------------------------- | :-------- | :------------------------- | :------------------------------------------------------ |
+| Azure Activity Log | Access and environment changes in Azure | No                                          | Subscription > Activity Logs | Yes       | Yes                        |                                                         |
+| pdh*-functionapp   | Application logs                        | TBD                                         | In Application Insights      | No        | ?                          |                                                         |
+| pdh*-pgsql         | Database errors, access errors, etc     | TBD                                         | Resource > Logs              | No        | ?                          | Could be other types of logs to capture like slow query |
+| pdh*-sftpserver    | ?                                       | TBD                                         | Resource > Container > Logs  | No        | ?                          | Is this even in use?                                    |
+| Front Door         | Network requests                        | TBD (Do we have PII/PHI as request params?) | Disabled                     | No        | ?                          | We should enable these logs                             |
+| WAF                | Firewall denials                        | No                                          | No WAF enabled, so no logs   | No        | ?                          |                                                         |
+| primedatahub*      | Access errors, etc                      | No                                          | Resource > Logs              | No        | ?                          |                                                         |
+
+
+## Potential PII/PHI in Application Logs
+
+1. Batch function logs entire message when received.
+2. Send function logs retry and failed messages.
+3. Transports can log failures of files. Any concerns of PII in them?
```

Comment:
OK, so I went off on the following wild goose chase:   Knowing that sftp fails in Test, I went to recreate a failure there, so I could answer this question with actual text from a log.  This led me to see that sftp was never happening in Test, which led me to discover a bug in our newest release.  So - good work!   You helped find a major bug!  ;)

Reply:
Digging through the source for `net.schmizz.sshj` it looks like the `IOException` thrown by the `SftpClient` logs the file name, but nothing else.

The other concern would be the connect method for the `java.net.Socket` method. That might merit some additional review.

Reply:
Haha, glad I could help find that @jimduff-usds.